README.md

```diff
@@ -0,0 +1,148 @@
+#### This application has a number of controllers that have the appropriate functionality:
+ 1. Authentication Controller:
+    - register new user
+    - login as user or admin
+ 2. Book Controller
+    - find all books
+    - get book by id
+    - create book
+    - delete book
+    - update book
+3. Category Controller
+   - create category
+   - get all category
+   - update category
+   - delete category
+   - get books by category
+4. Order Controller
+   - create order
+   - find all orders
+   - find all items in specific order
+   - find specific item in specific order
+   - update order
+5. Shopping Cart Controller
+   - create cart
+   - get cart
+   - update cart 
+   - delete cart
+## API  Testing with Postman
+![](media/postman.png)
+
+We have provided a Postman collection that you can use to test the API endpoints easily.
+This collection helps you set up contract tests to ensure that two separate systems are compatible and can communicate with one another.
+Follow these steps to get started:
+1. **Download Postman**: If you don't have Postman installed, you can [download it here](https://www.postman.com/downloads/).
+
+2. **Import the Collection**:
+
+    - Click the "Import" button in Postman.
+    - Select the downloaded Postman collection file: [Postman_collection](media/postmanCollection.json).
+
+3. **Configure Environment**:
+
+    - Create a new environment in Postman and set the base URL to your local or deployed API URL.
+    - You can use the default values for most variables, but make sure to set the `token` variable to your authentication token if required.
+
+4. **Run the Requests**:
+
+    - Open the imported collection in Postman.
+    - Select the desired request.
+    - Click the "Send" button to execute the request.
+    - Review the response to ensure everything is working as expected.
+
+5. **Explore Endpoints**:
+
+    - The collection includes a variety of requests to different endpoints. Feel free to explore and test them to understand the API's capabilities.
+
+6. **Note**: Some requests may require authentication. Ensure you are logged in and have the necessary permissions.
+
+## Contribution
+if you have a desire to participate in the improvement of the application, you can perform the following steps:
+1. Fork the [GitHub repository](https://github.com/OleksandrSaven/book-store) to your own account.
+
+2. Create a new branch for your feature or bug fix.
+
+3. Make your changes in your branch.
+
+4. Test your changes thoroughly to ensure they do not introduce new issues.
+
+5. Submit a pull request to the [main repository](https://github.com/OleksandrSaven/book-store) following the template provided.
+
+6. Your pull request will be reviewed and any necessary changes will be requested.
+
+7. Once your pull request is approved, it will be merged into the project.
+
+
+   Thank you for your interest in contributing to online-book-store! We appreciate your help in making the project better for everyone.

```

Comment:
Please avoid using 'We'. 'We' means that you worked on the project with someone else